### PR TITLE
Update cosmic_comp.ftl PL translation

### PR DESCRIPTION
**Pull request by VandaLHJ**
The idea behind sticky don't work well in my language and is probably misleading about the functionality. I opted for zakrywające okna, because it's always on top, but maybe someone will find better allegory for that kind of functionality later on. Also, stack isn't really good to translate, i opted for karta(tab), because it's what the stack is. It might be not the best translation, but i feel it's more natural.

### CODE REVIEW

File: resources/i18n/pl/cosmic_comp.ftl [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 window-menu-screenshot = Zrób zrzut ekranu
 window-menu-move = Przesuń
 window-menu-resize = Zmień rozmiar
 window-menu-move-prev-workspace = Przenieś do poprzedniego obszaru roboczego
 window-menu-move-next-workspace = Przenieś do następnego obszaru roboczego
-window-menu-stack = Włącz zakrywanie okien
-window-menu-unstack-all = Usuń zakrywanie okien
-window-menu-unstack = Usuń zakrycie okna
-window-menu-sticky = Przylepne okno
+window-menu-stack = Stwórz okna w kartach
```
Comment (Kartonrealista) [inline resources/i18n/pl/cosmic_comp.ftl:14]: Zgrupuj okna w karty

File: resources/i18n/pl/cosmic_comp.ftl [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+window-menu-unstack-all = Rozdziel okna w kartach
```
Comment (Kartonrealista) [inline resources/i18n/pl/cosmic_comp.ftl:15]: Rozgrupuj okna

File: resources/i18n/pl/cosmic_comp.ftl [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+window-menu-unstack = Rozdziel okno od kart
```
Comment (Kartonrealista) [inline resources/i18n/pl/cosmic_comp.ftl:16]: Oddziel okno od grupy

Comment (Kartonrealista) [reply resources/i18n/pl/cosmic_comp.ftl:16]: alt: Wydziel okno z grupy

File: resources/i18n/pl/cosmic_comp.ftl (hunk 1 above)
Comment (VandaLHJ) [reply resources/i18n/pl/cosmic_comp.ftl:14]: Yep, feels better than my fast pick.

**Comment by Kartonrealista** [comment]
I've made so suggestions based on terminology used in Polish versions of graphics and spreadsheet programs, where joining separate entities is mostly referred to as grouping. I don't think "Stwórz" sounds right, you're not creating something new, and a more typical word in computing is "Utwórz".

File: resources/i18n/pl/cosmic_comp.ftl (hunk 2 above)
Comment (VandaLHJ) [reply resources/i18n/pl/cosmic_comp.ftl:15]: I'm not too sure about this one, perhaps Rozdziel grupę okien would feel better?

File: resources/i18n/pl/cosmic_comp.ftl (hunk 3 above)
Comment (VandaLHJ) [reply resources/i18n/pl/cosmic_comp.ftl:16]: Yep, alt suggestion feels imo bit better.

File: resources/i18n/pl/cosmic_comp.ftl (hunk 2 above)
Comment (Kartonrealista) [reply resources/i18n/pl/cosmic_comp.ftl:15]: Sure

**Review by Drakulix — APPROVED** [review]
Thanks!

**Pull request merged**